tests/bsim/bluetooth/mesh/_mesh_test.sh

```diff
@@ -68,8 +70,9 @@ function RunTest(){
     Execute \
       ${exe_name} \
       -v=${verbosity_level} -s=$s_id -d=$idx -sync_preboot -RealEncryption=1 \
-      -testid=$testid ${testids["${testid}"]} ${test_options}
+      -rs=$seed -testid=$testid ${testids["${testid}"]} ${test_options}
     let idx=idx+1
+    let seed=seed+1
```

Comment:
Could you elaborate why this change is needed? Will mesh tests work without it?

Reply:
pb_remote_reprovision.sh   was failing in this PR.. Now I see that other tests are failing when devices have different random seed values.

Debuging with @m-alperen-sener, we were able to pass  pb_remote_reprovision.sh by having a small (2ms) delay in relay_send.

Instead of adding delay, we decided to have random seed value in test script

Reply:
We used to add/increase delays in a particular test in case when the issue is not caused by invalid implementation (for example: https://github.com/zephyrproject-rtos/zephyr/commit/e5fcca6e99a52334c0c726dfc1b0c432f8d944bb#diff-7df14fc51b85bcb032356d6314ea93067acb2b55ad69b8585edd8821709da43c). Changing the seed will affect other tests.

Reply:
I can have a look later today

Reply:
Please try this patch:
```
diff --git a/tests/bsim/bluetooth/mesh/src/test_provision.c b/tests/bsim/bluetooth/mesh/src/test_provision.c
index 171f8cccbb..36e3c1250a 100644
--- a/tests/bsim/bluetooth/mesh/src/test_provision.c
+++ b/tests/bsim/bluetooth/mesh/src/test_provision.c
@@ -554,26 +554,26 @@ static void node_configure_and_reset(void)
                                          &status));
        ASSERT_EQUAL(0, status);
 
-       k_sleep(K_SECONDS(1));
+       k_sleep(K_SECONDS(2));
 
        ASSERT_OK(bt_mesh_cfg_cli_mod_app_bind(0, current_dev_addr, current_dev_addr, 0x0,
                                           BT_MESH_MODEL_ID_HEALTH_SRV, &status));
        ASSERT_EQUAL(0, status);
 
-       k_sleep(K_SECONDS(1));
+       k_sleep(K_SECONDS(2));
 
        ASSERT_OK(bt_mesh_cfg_cli_mod_sub_add(0, current_dev_addr, current_dev_addr, 0xc000,
                                          BT_MESH_MODEL_ID_HEALTH_SRV, &status));
        ASSERT_EQUAL(0, status);
 
-       k_sleep(K_SECONDS(1));
+       k_sleep(K_SECONDS(2));
 
        ASSERT_OK(bt_mesh_cfg_cli_mod_pub_set(0, current_dev_addr, current_dev_addr,
                                          BT_MESH_MODEL_ID_HEALTH_SRV, &healthpub,
                                          &status));
        ASSERT_EQUAL(0, status);
 
-       k_sleep(K_SECONDS(1));
+       k_sleep(K_SECONDS(2));
 
        ASSERT_OK(bt_mesh_cfg_cli_node_reset(0, current_dev_addr, (bool *)&status));
```

Reply:
thanks will try it and get back on Monday